client/src/pages/pitchDoc/index.tsx

```diff
@@ -214,16 +214,11 @@ const PitchDoc = (): ReactElement => {
           <h3>Filters: </h3>
         </div>
         <div className="wrapper">
-          <MultiSelect
-            value={interests}
+          <InterestsSelect
+            values={interests}
             onChange={(values) =>
               setInterests(values ? values.map((item) => item.value) : [])
```

Comment:
this values check is probably needed but I can't figure out why. why couldn't we do 
`setInterests(values.map((item) => item.value))`

Reply:
I actually copied this from some react select documentation so I assumed that values would be undefined or null if nothing was selected but I just took it out and it seems to be working fine